ocp-resources/e2e/cluster-logging-install.yaml

```diff
@@ -50,6 +50,7 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging
 spec:
+  channel: stable-6.0
```

Comment:
This will be relevant too I think:

https://docs.openshift.com/container-platform/4.16/observability/logging/logging-6.0/log6x-upgrading-to-6.html